Update chains of trust and staging documenation (#1688)
letsencrypt/website: content/en/docs/staging-environment.md

* **Pretend Pear X1**
  * Subject: `O = (STAGING) Internet Security Research Group, CN = (STAGING) Pretend Pear X1`
  * Key type: `RSA 4096`
  * Certificate details: [der](/certs/staging/letsencrypt-stg-root-x1.der), [pem](/certs/staging/letsencrypt-stg-root-x1.pem), [txt](/certs/staging/letsencrypt-stg-root-x1.txt)
* **Bogus Broccoli X2**
  * Subject: `O = (STAGING) Internet Security Research Group, CN = (STAGING) Bogus Broccoli X2`
  * Key type: `ECDSA P-384`
  * Certificate details (self-signed): [der](/certs/staging/letsencrypt-stg-root-x2.der), [pem](/certs/staging/letsencrypt-stg-root-x2.pem), [txt](/certs/staging/letsencrypt-stg-root-x2.txt)
  * Certificate details (cross-signed by Pretend Pear X1): [der](/certs/staging/letsencrypt-stg-root-x2-signed-by-x1.der), [pem](/certs/staging/letsencrypt-stg-root-x2-signed-by-x1.pem), [txt](/certs/staging/letsencrypt-stg-root-x2-signed-by-x1.txt)

## Subordinate (Intermediate) CAs

> The staging environment has four active intermediate certificates, issued from the untrusted roots detailed above. Currently, these are:

Reviewer: ```suggestion
The staging environment has intermediate certificates that mimic production, issued from the untrusted roots detailed above. Like in production, not all are in use at any time. The full list of current intermediates is:
```

Now this calls out for actives but doesn't say what they are. I think we can just leave that unspecified.

Author: Ah good catch, thanks.

> The staging environment has a certificate hierarchy that [mimics production](/certificates).

Reviewer: ```suggestion
The staging environment has a certificate hierarchy that [mimics production](/certificates). The names have been modified with a prefix of (STAGING) and unique name to make them clearly distinct from their production counterparts.
```

* **Pretend Pear X1**
  * Subject: `O = (STAGING) Internet Security Research Group, CN = (STAGING) Pretend Pear X1`
  * Key type: `RSA 4096`
  * Certificate details: [der](/certs/staging/letsencrypt-stg-root-x1.der), [pem](/certs/staging/letsencrypt-stg-root-x1.pem), [txt](/certs/staging/letsencrypt-stg-root-x1.txt)
* **Bogus Broccoli X2**
  * Subject: `O = (STAGING) Internet Security Research Group, CN = (STAGING) Bogus Broccoli X2`
  * Key type: `ECDSA P-384`
  * Certificate details (self-signed): [der](/certs/staging/letsencrypt-stg-root-x2.der), [pem](/certs/staging/letsencrypt-stg-root-x2.pem), [txt](/certs/staging/letsencrypt-stg-root-x2.txt)
  * Certificate details (cross-signed by Pretend Pear X1): [der](/certs/staging/letsencrypt-stg-root-x2-signed-by-x1.der), [pem](/certs/staging/letsencrypt-stg-root-x2-signed-by-x1.pem), [txt](/certs/staging/letsencrypt-stg-root-x2-signed-by-x1.txt)

## Subordinate (Intermediate) CAs

> The staging environment has four active intermediate certificates, issued from the untrusted roots detailed above. Currently, these are:

Reviewer: I think we should list the inactive entries too (we're including them in static/ after all)

> * (STAGING) Psuedo Plum E5
> * (STAGING) False Fennel E6
> * (STAGING) Counterfeit Cashew R10
> * (STAGING) Wannabe Watercress R11

Reviewer: ```suggestion
* (STAGING) Pseudo Plum E5
* (STAGING) False Fennel E6
* (STAGING) Puzzling Parsnip E7
* (STAGING) Mysterious Mulberry E8
* (STAGING) Fake Fig E9
* (STAGING) Counterfeit Cashew R10
* (STAGING) Wannabe Watercress R11
* (STAGING) Riddling Rhubarb R12
* (STAGING) Tenuous Tomato R13
* (STAGING) Not Nectarine R14
```
